tests/acceptance/features/apiSpaces/moveSpaces.feature

```diff
@@ -0,0 +1,234 @@
+    When user "Alice" moves file "project1.txt" from space "Project1" to "project1.txt" inside space "Project2" using the WebDAV API
+    Then the HTTP status code should be "<https_status_code>"
+    And for user "Alice" the space "Project1" should contain these entries:
+      | project1.txt       |
+    And for user "Alice" the space "Project2" should not contain these entries:
+      | project1.txt       |
+    Examples:
+      | from_role | to_role | https_status_code |
+      | manager   | manager | 502               |
+      | editor    | manager | 502               |
+      | manager   | editor  | 502               |
+      | editor    | editor  | 502               |
+      | manager   | viewer  | 403               |
+      | editor    | viewer  | 403               |
+      | viewer    | manager | 403               |
+      | viewer    | editor  | 403               |
+      | viewer    | viewer  | 403               |
+
+
+  Scenario Outline: User moves a file from a space project with different role to a space personal
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has uploaded a file inside space "Project" with content "Project content" to "project.txt"
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    When user "Alice" moves file "project.txt" from space "Project" to "project.txt" inside space "Personal" using the WebDAV API
+    Then the HTTP status code should be "<https_status_code>"
+    And for user "Alice" the space "Project" should contain these entries:
+      | project.txt       |
+    And for user "Alice" the space "Personal" should not contain these entries:
+      | project.txt       |
+    Examples:
+      | role    | https_status_code |
+      | manager | 502               |
+      | editor  | 502               |
+      | viewer  | 403               |
+
+
+  Scenario Outline: User moves a file from space project with different role to space shares jail with different role (permission)
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded a file inside space "Project" with content "Project content" to "project.txt"
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permissions>"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" moves file "project.txt" from space "Project" to "/testshare/project.txt" inside space "Shares Jail" using the WebDAV API
```

Comment:
"Shares Jail" never gets mentioned before here in the scenario. It is a bit surprising - there is a special space name "Shares Jail" that already/always exists. This is already in many other scenarios, so it is not a new thing.

We should write something somewhere about this "behavior". I will make a separate issue or PR - `listSpaces.feature` looks like the place to make a note about this "magic".